tests/test_server.py

```diff
@@ -287,3 +287,40 @@ def test_get_identities(
     assert response.status_code == 200
     assert data["traits"] == []
     assert data["flags"]
+
+
+@pytest.mark.parametrize(
+    "environment_key,expected_status",
+    [
+        ("ser.good", 200),
+        ("ser.bad", 401),
+        (None, 401),
+    ],
+)
```

Comment:
I'd rather write these as separate tests. I don't like the fact that we're hard coding the string twice and using conditional logic in the test based on the parameterised values.